invent-code-1/main.cpp

```diff
@@ -7,6 +7,12 @@ int main() {
   string city;
 
   // Step 1. Ask the user for their name and their city.
+  cout << "What is your name, bigshot?" << endl;
+  cin >> name;
+
+  cout << "From where do you hail?" << endl;
+  cin >> city; 
 
   // Step 2. Print the greeting back to the user.
+  cout << "Welcome to " << city << "! Wash your hands, don't touch your face, let's keep " << city << " a COVID-free place." << endl;
```

Comment:
This is a great greeding :D!!  You didn't use the variable "name" in the final greeding though.

Reply:
Fixed.